packages/radix-ui-themes/src/styles/tokens/color.css

```diff
@@ -202,8 +202,8 @@
   & ::selection {
     background-color: var(--accent-a6);
   }
-  & [data-accent-color='gray']::selection,
-  & [data-accent-color='gray'] ::selection {
+  & :where([data-accent-color='gray'])::selection,
+  & :where([data-accent-color='gray']) ::selection {
```

Comment:
Same question about whether we need a complex selector for the gray accent here

Reply:
Pretty sure that was needed for nested sections…

Reply:
Simplified this selector https://github.com/radix-ui/themes/pull/79/commits/30baf1b029e146aa47b20b2a65d0376df09173d2